ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,19 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace Primenumberchecker {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number:");
+         int number = int.Parse (Console.ReadLine ());
+         bool isPrime = true;
+         for (int i = 2; i < number; i++) {

```

Comment:
Do we have to do the divisibility check till number - 1 ? Try the Wikipedia article